server/include/localization.hpp

```diff
@@ -20,22 +27,26 @@ class Localization
 {
 public:
 #ifdef ENABLE_TESTS
-    friend class LocalizationFixture;
+	friend class LocalizationFixture;
```

Comment:
Please let's be civil here and use spaces instead of tabs! :grin:

Reply:
Same goes for the rest of this file.

Reply:
...tabs are man's best friend. :grinning: